.gitmodules

```diff
@@ -23,8 +23,9 @@
 	path = contrib/capnproto
 	url = https://github.com/ClickHouse/capnproto
 [submodule "contrib/double-conversion"]
+	branch = "ch-loongarch"
 	path = contrib/double-conversion
-	url = https://github.com/google/double-conversion
+	url = https://github.com/qiangxuhui/double-conversion
```

Comment:
Can we instead please create a fork of double-conversion in ClickHouse's GitHub organization? (https://github.com/ClickHouse/). At last that would be the standard practice.

Reply:
I have sended a PR， https://github.com/ClickHouse/double-conversion/pull/1/files.

Reply:
Thanks. Please see my comment [here](https://github.com/ClickHouse/double-conversion/pull/1). Additionally, we'd need to switch to `url = https://github.com/ClickHouse/double-conversion`. This can be done by (1) `git rm contrib/double-conversion` + committing the result, followed by (2) `cd contrib/`, `git submodule add --force https://github.com/ClickHouse/double-conversion.git` + committing the result, followed by (3) `cd double-conversion`, `git switch --create clickhouse origin/clickhouse`, `cd ..`, followed by a commit.

EDIT: This seems to get complicated. I'd be good with keeping `url = https://github.com/qiangxuhui/double-conversion` for now. When I am back from vacation, I can fix this up.

Reply:
Fixed: https://github.com/ClickHouse/ClickHouse/pull/62177